content/issues/2023-04-26-scheduled-db-cleanup.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Scheduled DB Cleanup"
+date: "2023-04-26T07:00:00+02:00"
+affected:
+  - API
+  - Dashboard
+resolved: false
```

Comment:
```suggestion
```

see [Jirka's comment](https://github.com/packit/status/pull/71#discussion_r1157461403)

Reply:
Thank you, fixed, and updated the todo-list to make this clear.